backend/routes/profile.py

```
@@ -0,0 +1,93 @@
+from typing import List
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)